operator/controller/src/test/resources/podtemplatespec-test-cases.yaml

```diff
@@ -0,0 +1,481 @@
+cases:
+
+  - id: 'Empty 1'
```

Comment:
I left them out because as currently implemented, these are taken directly from the spec and do not need to be merged from the factory, and any additional changes that the operator does will be done on top of that.

Reply:
as far as we remember to implement the relevant checks when implementing new features, I'm ok with this logic.